lib/galaxy/webapps/config_manage.py

```diff
@@ -106,6 +106,11 @@
         'default': '$uwsgi_module',
         'type': 'str',
     }),
+    ('thunder-lock', {
+        'desc': """It is usually a good idea to set this to ``true`` if processes is greater than 1.""",
+        'default': 'false',
+        'type': 'str',
```

Comment:
These could be changed to:
```
        'default': False,
        'type': 'bool',
```
after https://github.com/galaxyproject/galaxy/pull/5747 is merged.